docs/csharp/language-reference/builtin-types/collection-expressions.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Collection expressions"
```

Comment:
As it says "expressions", should it be then at the *operators* folder and under "Operators and expressions" TOC?

Reply:
I'm open to that idea. Where in the operators and expressions section would you add it?

Reply:
It looks to deserve its own page (like it's already now), so it would go somewhere here (to match alphabetical order):
![image](https://github.com/dotnet/docs/assets/15279990/dc0609a0-45b3-4dee-a3a0-4602acf391d5)

Reply:
I'm not making this change yet, but I think it is useful.

Note to self: If I do this, the updated articles for *collections* and *arrays* should add links to the collection expressions article. It will be a bit farther away in the TOC, and links would be helpful.

Note to reviewers: I'd like other opinions on the location of the new article: Should it stay in the section on built-in types, or move to the operators and expressions section?